contract/test/orca-contract.test.js

```diff
@@ -37,237 +79,127 @@ const test = anyTest;
+        },
+        executeTx() {
+          throw Error('mock TODO makeexecuteTxAccount');
         },
-        getChain: async name => {
-          if (
-            name === 'agoric' ||
-            name === 'osmosis' ||
-            name === 'agoriclocal' ||
-            name === 'osmosislocal'
-          ) {
-            const state = harden({
-              name,
-              chainId: `${name}local`,
-              denom: name === 'agoric' ? 'ubld' : 'uosmo',
-              expectedAddressPrefix: name === 'agoric' ? 'agoric' : 'osmo',
-              details: `${name} chain details`,
-              stakingTokens: [{ denom: name === 'agoric' ? 'ubld' : 'uosmo' }],
-            });
-
-            return harden({
-              ...state,
-              makeAccount: () =>
-                Far('Account', {
-                  getChainId: () => state.chainId,
-                  getAccountAddress: () => `${state.name}AccountAddress`,
-                  getAddress: () =>
-                    harden({
-                      chainId: state.chainId,
-                      value: `${state.name}AccountAddress`,
-                      encoding: 'bech32', // or 'ethereum', based on your requirements
-                    }),
-                  getBalance: () => `1000${state.denom}`,
-                }),
-              getChainInfo: () =>
-                Far('ChainInfo', {
-                  getChainId: () => state.chainId,
-                  getDenom: () => state.denom,
-                  getExpectedAddressPrefix: () => state.expectedAddressPrefix,
-                }),
-            });
-          }
-          throw Error(`chain not found: ${name}`);
+        async monitorTransfers(tap) {
+          /** @type {TargetRegistration} */
+          const reg = Far('TR mock', {
+            revoke() {
+              throw Error('TODO revoke');
+            },
+            updateTargetApp() {
+              throw Error('TODO updateTargetApp');
+            },
+          });
+          return reg;
         },
-      };
-      return chainHub;
+      });
+      return acct;
     },
-    makeAccount: async name => {
-      const chainHub = await E(cosmosInterchainService).getChainHub();
-      const chain = await E(chainHub).getChain(name);
-      return E(chain).makeAccount();
+    query() {
+      throw Error('mock TODO query');
     },
-    getChainInfo: async name => {
-      const chainHub = await E(cosmosInterchainService).getChainHub();
-      const chain = await E(chainHub).getChain(name);
-      return E(chain).getChainInfo();
+    queryMany() {
+      throw Error('mock TODO queryMany');
     },
   });
 
+  const vowTools = prepareVowTools(powers.zone);
+
   return {
-    zoe,
     bundle,
     bundleCache,
-    feeMintAccess,
     cosmosInterchainService,
-    agoricNames,
-    storageNode: makeDummyStorageNode(),
-    marshaller: makeDummyMarshaller(),
+    localchain,
+    vowTools,
+    consume: powers.consume,
```

Comment:
Seems leaky to put this CoreEval thing into the context.

Consider calling it "bootstrapSpace"